packages/editor/README.md

```diff
@@ -401,7 +401,7 @@ _Parameters_
 
 -   _props_ `Object`: The component props.
 -   _props.close_ `Function`: The function to close the dialog.
--   _props.renderDialog_ `Function`: The function to render the dialog.
+-   _props.renderDialog_ `boolean`: Whether to render the component with modal dialog behavior.
```

Comment:
Would this qualify as a breaking change?

(sorry for the delayed comment, I'm still catching up with notifications from the past weeks)

Reply:
No, the documentation was wrong when added. The value was always boolean.

Reply:
IMO, this component shouldn't be public, but that ship has sailed :(